make/tools.mk

```diff
@@ -565,15 +565,14 @@ endif
 
 # OPENSSL download URL
 ifdef WINDOWS
-  openssl_install: OPENSSL_URL  := http://slproweb.com/download/Win32OpenSSL-1_0_2c.exe
+  openssl_install: OPENSSL_URL  := https://slproweb.com/download/Win32OpenSSL-1_0_2d.exe
```

Comment:
Thanks.

P.S. I get where you're coming from. There's a good philosophical question about whether every git commit should be able to compile and work, or whether git commits which break-- but then fix-- the build are okay.

I tend to want to accomplish the former, but I don't hesitate to do the latter. In my commit messages, you'll find lots of ```*** DOES NOT COMPILE***```. It seems to work for me, but to each his own...